torch/overrides.py

```diff
@@ -1785,6 +1787,37 @@ def _no_torch_function_mode() -> Iterator[None]:
         _set_torch_function_mode(old)
 
 
+class TorchFunctionModeInfo(_ModeInfo):
+    def __init__(self):
+        super().__init__(mode_name="torch_function", mode_class=TorchFunctionMode,
+                         base_mode_class=BaseTorchFunctionMode,
+                         required_fn="__torch_function__")
+
+    def is_allowed_type(self, mode) -> bool:
+        return mode is None or\
```

Comment:
wow, I'm surprised the linter didn't complain about backslash after keyword with no intervening space lol